src/dashboard/frontend/src/components/CommandDeck/WorktreePickerMenu.module.css

```
@@ -0,0 +1,263 @@
+}
+
+.input {
+  font: inherit;
+  font-size: 12px;
+  padding: 5px 8px;
+  border-radius: 4px;
+  border: 1px solid var(--border);
+  background: var(--background);
+  color: var(--foreground);
+  font-variant-numeric: tabular-nums;
+  font-family: "SF Mono", "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)